Sources/Common/DataModel/Quad/test/testQuad.js

```diff
@@ -0,0 +1,71 @@
+import test from 'tape-catch';
+import vtkQuad from 'vtk.js/Sources/Common/DataModel/Quad';
+import vtkPoints from 'vtk.js/Sources/Common/Core/Points';
+
+test('Test vtkQuad instance', (t) => {
+  t.ok(vtkQuad, 'Make sure the class definition exists');
+  const instance = vtkQuad.newInstance();
+  t.ok(instance);
+  t.end();
+});
+
+test('Test vtkQuad intersectWithLine flat', (t) => {
+  const yesIntersection = 1;
+  const noIntersection = 0;
```

Comment:
Please add a `Constants.js` like the one at `'vtk.js/Sources/Common/DataModel/Line/Constants';`. Name the enum `QuadWithLineIntersectionState` and use `QuadWithLineIntersectionState.NO_INTERSECTION` and `QuadWithLineIntersectionState.YES_INTERSECTION` instead of these constants.